nixos/lib/test-driver/test_driver/driver.py

```diff
@@ -55,6 +55,7 @@ def __init__(
         tmp_dir = get_tmp_dir()
 
         with rootlog.nested("start all VLans"):
+            vlans = list(dict.fromkeys(vlans))
```

Comment:
oh...interesting. good point!

what do you think about:

```python
vlans = list(set(vlans))
```

Reply:
That sounds even better. I'll make the change.